src/transformers/modeling_bert.py

```diff
@@ -935,6 +1048,7 @@ def forward(
                 DeprecationWarning,
             )
             labels = kwargs.pop("masked_lm_labels")
+        assert "lm_labels" not in kwargs, "Use `BertWithLMHead` for autoregressive language modeling task."
```

Comment:
It wouldn't be in the kwargs as it's still in the method signature

Reply:
Good point, forgot to remove that arg from the signature.